services/QuillLMS/db/external/bigquery/mat_views/recent_reporting_sessions.sql

```diff
@@ -0,0 +1,47 @@
+CREATE MATERIALIZED VIEW analytics-data-stores.lms.recent_reporting_sessions
+OPTIONS (
+  enable_refresh = true, refresh_interval_minutes = 1440,
```

Comment:
We want this to auto refresh off peak hours, to minimize downtime.

Reply:
Maybe I'm not understanding, but how is this refreshing off peak hours? We need the stats to be up to date for the day before, but I'm not sure how this setting ensures that. I'm reading this as refresh every 24-hours with a max staleness of 24 hours. Could this refresh at noon one day and someone looking at in the next morning wouldn't have yesterday's full stats?